crates/bevy_render/src/mesh/mesh/mod.rs

```diff
@@ -566,36 +589,16 @@ impl Mesh {
 
         match self.indices() {
```

Comment:
I think we could unify the 2 branches of this match, as they do the exact same operation but either on an index list, or on `(0..positions.len())`, but this is nitpicking